app/src/main/java/org/dhis2/utils/customviews/MoreMenuView.kt

```diff
@@ -39,3 +43,8 @@ fun <T> MoreOptionsWithDropDownMenuButton(
         onItemClick(itemId)
     }
 }
+
+inline val Double.dw: Dp
```

Comment:
why do we need this function? should we extract it to some place to reuse it in other places?

Reply:
It is just a way to calculate the percentage for the max width. I don't think it needs to be on the mobile-ui, but could be useful to move it to commons.